### DISCUSSION TITLE
data-augmentation / Increase number of datas

### DISCUSSION THREAD

**Discussion by product-imki** [facebookresearch/detectron2#2591] [Q&A]
## 📚 Documentation
Hello everyone,
There are some points that I do not understand when using data-augmentation with detectron2.
I have created my own coco datasets using cocoanotator and they are composed of 68 images for the train and 10 images for validation with their respective json file. I wrote my own mapper to use the different transformations available (flip and rotate for example) and I wrote my own Trainer. I'm not sure if there is a setting to adjust the amount of data to be used for training and testing. I mean is it possible to adapt the number of call of the dataloader during the training to successively load 3000 images instead of 68?
The idea is to pass 3000 images instead of 68 for one epoch and artificially increase the number of data.

**Comment by ppwwyyxx** [comment]
> The idea is to pass 3000 images instead of 68 for one epoch and artificially increase the number of data.

There is no concept of epoch in detectron2. You can train for 999999 iterations and it will just use your data for many times.

**Reply by product-imki** [reply]
Thank you for your quick reply ! I was not sure if it was actually the case or not
